lib/ex_doc/formatter/html/templates.ex

```diff
@@ -169,6 +169,13 @@ defmodule ExDoc.Formatter.HTML.Templates do
   end
 
   defp sidebar_items_node(module_node) do
+    escape_quotes = &String.replace(&1, "\"", "\\\"")
```

Comment:
Might be good to escape `\` too in case it appears at the end of a module name (unlikely though not impossible)

Reply:
Instead of `escape`, I think we should call `#{inspect(id)}` instead of `#{id}` when generating the sidebar data in the first place. :) That should take care of everything.

Reply:
Another option is to call `html_escape` or similar.